metcalcpy/util/ecnt_statistics.py

```diff
@@ -404,6 +410,124 @@ def calculate_ecnt_spread_plus_oerr(input_data, columns_names, aggregation=False
+        than their observations
+        Args:
+            input_data: 2-dimensional numpy array with data for the calculation
+                1st dimension - the row of data frame
+                2nd dimension - the column of data frame
+            columns_names: names of the columns for the 2nd dimension as Numpy array
+            aggregation: if the aggregation on fields was performed
+
+        Returns:
+            calculated N_LT_OBS as float
+            or None if some data values are missing or invalid
+    """
+    warnings.filterwarnings('error')
+    try:
+        n_lt_obs = sum_column_data_by_name(input_data, columns_names, 'n_lt_obs')
+        result = round_half_up(n_lt_obs, PRECISION)
+    except (TypeError, ZeroDivisionError, Warning, ValueError):
+        result = None
+    warnings.filterwarnings('ignore')
+    return result
+
+def calculate_ecnt_me_ge_obs(input_data, columns_names, aggregation=False):
+    """Performs calculation of ME_GE_OBS - The Mean Error of the ensemble values
+        greater than or equal to their observations
+        Args:
+            input_data: 2-dimensional numpy array with data for the calculation
+                1st dimension - the row of data frame
+                2nd dimension - the column of data frame
+            columns_names: names of the columns for the 2nd dimension as Numpy array
+            aggregation: if the aggregation on fields was performed
+
+        Returns:
+            calculated ME_GE_OBS as float
+            or None if some data values are missing or invalid
+    """
+    warnings.filterwarnings('error')
+    try:
+        n_ge_obs = sum_column_data_by_name(input_data, columns_names, 'n_ge_obs')
+        me_ge_obs = sum_column_data_by_name(input_data, columns_names, 'me_ge_obs')/n_ge_obs
+        result = round_half_up(me_ge_obs, PRECISION)
+    except (TypeError, ZeroDivisionError, Warning, ValueError):
+        result = None
+    warnings.filterwarnings('ignore')
+    return result
+
+def calculate_ecnt_me_lt_obs(input_data, columns_names, aggregation=False):
+    """Performs calculation of ME_GE_OBS - The Mean Error of the ensemble values
+        greater than or equal to their observations
+        Args:
+            input_data: 2-dimensional numpy array with data for the calculation
+                1st dimension - the row of data frame
+                2nd dimension - the column of data frame
+            columns_names: names of the columns for the 2nd dimension as Numpy array
+            aggregation: if the aggregation on fields was performed
+
+        Returns:
+            calculated ME_GE_OBS as float
+            or None if some data values are missing or invalid
+    """
+    warnings.filterwarnings('error')
+    try:
+        n_lt_obs = sum_column_data_by_name(input_data, columns_names, 'n_lt_obs')
+        me_lt_obs = sum_column_data_by_name(input_data, columns_names, 'me_lt_obs')/n_lt_obs
```

Comment:
I don't think this logic is correct. We need to compute an aggregated ME_LT_OBS as a weighted average where N_LT_OBS defines the weight. That's different from what's being compute here.